syntax = "proto3";
package proto;

option go_package = "github.com/yourusername/go-ex/proto";

service UserService {
  rpc GetUser(UserRequest) returns (UserResponse) {}
  rpc CreateUser(CreateUserRequest) returns (UserResponse) {}
}

message UserRequest {
  int64 id = 1;
}

message CreateUserRequest {
  string name = 1;
  string email = 2;
}

message UserResponse {
  int64 id = 1;
  string name = 2;
  string email = 3;
}